tests/UnitTests/CoreGraphics.drawing/EntryPoint.cpp

```diff
@@ -82,6 +82,11 @@ class CommandLineDrawingTestConfigImpl : public DrawingTestConfigImpl {
     virtual std::string GetComparisonPath() override {
         return _comparisonPath;
     }
+
+    virtual std::string GetResourcePath(const std::string& resource) override {
+        // Using / as a path separator is valid on both Windows and OS X.
+        return __ModuleDirectory() + "/data/" + resource;
```

Comment:
> "/data/"  [](start = 36, length = 10)

Curious: In windows the files aren't copied into data. in OSX it is?
Also it would be better not to hard code /data/ in this function and pass it in?
(make it more generic? maybe?)